Initial apps

shawnadelic opened the pull request.

mfgryan commented:
Everything in app/* looks good. I was able to run following the README instructions.

Feedback:
- I think it make more sense to move everything in app/* up 1 level. Thoughts on this layout?
- Add .DSstore to the .gitignore. This is a Mac specific file.
- We should go ahead and move js, css, and images dirs over to the client dir so everything is in one place.

shawnadelic commented:
* I went ahead and moved everything up one level and merged the two README files together.
* Added .DS_Store to gitignore
* Added an `assets` directory for css, js, and image assets for current static site
